library/vtproto/v0.3.0/buf.plugin.yaml

```diff
@@ -0,0 +1,13 @@
+version: v1
+name: buf.build/library/vtproto
```

Comment:
We might want to name this `vtproto-go`? What do you think @mfridman / @smallsamantha? It might make it more discoverable and be similar to how we handle twirp.

Reply:
yup i agree @pkwarren i think the consistency is a good call

Reply:
For the most part, I agree. My only concern is we have some standardized way to do this. 

Maybe in lieu of a plugin name that includes the language we suffix it?